zhaquirks/ikea/__init__.py

```diff
@@ -7,13 +7,22 @@
 from zigpy.zcl.clusters.general import Scenes
 from zigpy.zcl.clusters.lightlink import LightLink
 
-from zhaquirks import DoublingPowerConfigurationCluster
+from zhaquirks import (
+    DoublingPowerConfigurationCluster,
+    EventableCluster,
+    PowerConfigurationCluster,
+)
 
 _LOGGER = logging.getLogger(__name__)
 
 IKEA = "IKEA of Sweden"
-IKEA_CLUSTER_ID = 0xFC7C  # decimal = 64636
 WWAH_CLUSTER_ID = 0xFC57  # decimal = 64599 ('Works with all Hubs' cluster)
+IKEA_CLUSTER_ID = 0xFC7C  # decimal = 64636
+IKEA_SHORTC_CLUSTER_V1_ID = 0xFC7F  # decimal = 64639 Shortcut V1 commands.
+IKEA_M_SWITCH_CLUSTER_ID = (
+    0xFC80  # decimal = 64640 Shortcut V2 / Matter Switch commands.
+)
```

Comment:
Hmm, it would be best if we could shorten this comment a bit, so black doesn't split it up (and maybe also remove the dots at the end of the two comments)